markdown/landing/index.md

```diff
@@ -48,4 +50,4 @@ Documentation and by extension headers/categories are a work in progress.
 - [trigger_setfire](../mapping/trigger_setfire.md)
 - [trigger_awardpoints](../mapping/trigger_awardpoints.md)
 - [trigger_interact](../mapping/trigger_interact.md)
-- [trigger_atroundend](../mapping/trigger_atroundend.md)
\ No newline at end of file
+- [trigger_atroundend](../mapping/trigger_atroundend.md)
```

Comment:
I have no clue why GitHub marked this as a change, I made no such change, maybe my editor auto-removed a space or something. But GitHub does not appear to show a trailing space soooo.....